burotel/indico_burotel/client/js/actions.js

```diff
@@ -0,0 +1,43 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2019 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import fetchRoomsLongTermAttributeURL from 'indico-url:plugin_burotel.rooms_long_term_attribute';
+
+import {ajaxAction} from 'indico/utils/redux';
+import {indicoAxios} from 'indico/utils/axios';
+
+
+export const FETCH_ROOMS_LONG_TERM_ATTRIBUTE_ERROR = 'rooms/FETCH_ROOMS_LONG_TERM_ATTRIBUTE_ERROR';
+export const FETCH_ROOMS_LONG_TERM_ATTRIBUTE_REQUEST = 'rooms/FETCH_ROOMS_LONG_TERM_ATTRIBUTE_REQUEST';
+export const FETCH_ROOMS_LONG_TERM_ATTRIBUTE_SUCCESS = 'rooms/FETCH_ROOMS_LONG_TERM_ATTRIBUTE_SUCCESS';
+export const ROOMS_LONG_TERM_ATTRIBUTE_RECEIVED = 'rooms/ROOMS_LONG_TERM_ATTRIBUTE_RECEIVED';
+
+
+function fetchRoomsLongTermAttribute() {
+    return async (dispatch) => {
```

Comment:
`return ajaxAction(...)`; - this extra wrapper is onyl needed if you need to access the store to get some params etc